cccatalog-api/cccatalog/api/serializers/image_serializers.py

```diff
@@ -357,8 +357,8 @@ class ImageSearchResultsSerializer(serializers.Serializer):
 class InputErrorSerializer(serializers.Serializer):
     """ Returned if invalid query parameters are passed. """
     detail = serializers.CharField()
-    fields = serializers.ListField()
-    error = serializers.CharField()
+    field = serializers.ListField()
```

Comment:
I think that instead of changing `fields` -> `field` in the documentation, we should do the opposite, i.e change the API response to have key `fields` instead of `field` because it is a list of query parameters that result in the 400 error and therefore can contain multiple entries.  (eg: https://api.creativecommons.engineering/v1/images?q=+&page_size=ss).

Tagging @aldenstpage for their opinion on this.

Reply:
Changing the api response might have some side effects such as the applications currently utilising the api will need to be updated too to handle the change.